tests/hw/esp-tester/tester.toit

```diff
@@ -156,6 +171,7 @@ class TestDevice:
     if port:
       port.close
       port = null
+    directory.rmdir --recursive tmp-dir
```

Comment:
Will this work if done a second time? I guess so.

Reply:
not yet...
I added a check.
done.